packages/dashboard-frontend/src/services/workspace-client/devworkspace/devWorkspaceClient.ts

```diff
@@ -396,6 +401,10 @@ export class DevWorkspaceClient extends WorkspaceClient {
     if (!workspace.spec.template.components) {
       workspace.spec.template.components = [];
     }
+    if (workspace.spec.template.components.some(component => component.name === pluginName)) {
```

Comment:
when it can happen? Who provision objects twice.
I assume it's not normal case, if so - it makes sense to log warning, or at least debug.

Reply:
I removed that, it was added during debugging and I forgot to get rid of it.